cli/tests/pcluster/templates/test_cdk_builder_utils.py

```diff
@@ -10,10 +10,238 @@
+                ec2.CfnLaunchTemplate.BlockDeviceMappingProperty(device_name="/dev/xvdbh", virtual_name="ephemeral7"),
+                ec2.CfnLaunchTemplate.BlockDeviceMappingProperty(device_name="/dev/xvdbi", virtual_name="ephemeral8"),
+                ec2.CfnLaunchTemplate.BlockDeviceMappingProperty(device_name="/dev/xvdbj", virtual_name="ephemeral9"),
+                ec2.CfnLaunchTemplate.BlockDeviceMappingProperty(device_name="/dev/xvdbk", virtual_name="ephemeral10"),
+                ec2.CfnLaunchTemplate.BlockDeviceMappingProperty(device_name="/dev/xvdbl", virtual_name="ephemeral11"),
+                ec2.CfnLaunchTemplate.BlockDeviceMappingProperty(device_name="/dev/xvdbm", virtual_name="ephemeral12"),
+                ec2.CfnLaunchTemplate.BlockDeviceMappingProperty(device_name="/dev/xvdbn", virtual_name="ephemeral13"),
+                ec2.CfnLaunchTemplate.BlockDeviceMappingProperty(device_name="/dev/xvdbo", virtual_name="ephemeral14"),
+                ec2.CfnLaunchTemplate.BlockDeviceMappingProperty(device_name="/dev/xvdbp", virtual_name="ephemeral15"),
+                ec2.CfnLaunchTemplate.BlockDeviceMappingProperty(device_name="/dev/xvdbq", virtual_name="ephemeral16"),
+                ec2.CfnLaunchTemplate.BlockDeviceMappingProperty(device_name="/dev/xvdbr", virtual_name="ephemeral17"),
+                ec2.CfnLaunchTemplate.BlockDeviceMappingProperty(device_name="/dev/xvdbs", virtual_name="ephemeral18"),
+                ec2.CfnLaunchTemplate.BlockDeviceMappingProperty(device_name="/dev/xvdbt", virtual_name="ephemeral19"),
+                ec2.CfnLaunchTemplate.BlockDeviceMappingProperty(device_name="/dev/xvdbu", virtual_name="ephemeral20"),
+                ec2.CfnLaunchTemplate.BlockDeviceMappingProperty(device_name="/dev/xvdbv", virtual_name="ephemeral21"),
+                ec2.CfnLaunchTemplate.BlockDeviceMappingProperty(device_name="/dev/xvdbw", virtual_name="ephemeral22"),
+                ec2.CfnLaunchTemplate.BlockDeviceMappingProperty(device_name="/dev/xvdbx", virtual_name="ephemeral23"),
+                ec2.CfnLaunchTemplate.BlockDeviceMappingProperty(
+                    device_name="/dev/sda1",
+                    ebs=ec2.CfnLaunchTemplate.EbsProperty(
+                        volume_size=10,
+                        encrypted=True,
+                        volume_type="mockVolumeType",
+                        iops=15,
+                        throughput=20,
+                        delete_on_termination=True,
+                    ),
+                ),
+            ],
+            id="test with all root volume fields populated",
+        ),
+        pytest.param(
+            RootVolume(
+                encrypted=True,
+                volume_type="mockVolumeType",
+                iops=15,
+                throughput=20,
+                delete_on_termination=True,
+            ),
+            "centos7",
```

Comment:
let's try with another os, in this way will validate that `device_name` for the root volume is a different one.
We should have a separate unit tests to verify all the OSes but we can cover something with this switch.

Reply:
Agreed, however note that this is not something that influences our logic (we use the values defined in our `constants.py` file for the `OS_MAPPING`).